docker/server-jetty/src/main/server-jetty/requirements.txt

```diff
@@ -1,19 +1,20 @@
-adbc-driver-manager==0.10.0
-adbc-driver-postgresql==0.10.0
+adbc-driver-manager==0.11.0
+adbc-driver-postgresql==0.11.0
 connectorx==0.3.2; platform.machine == 'x86_64'
 deephaven-plugin==0.6.0
-java-utilities==0.2.0
+importlib_resources==6.4.0
```

Comment:
What's this new dependency needed for?

Reply:
No idea. Something transitively depends on it now so we pick it up.